compiler/src/iree/compiler/Codegen/LLVMGPU/TransformExtensions/LLVMGPUExtensions.cpp

```diff
@@ -98,7 +98,7 @@ transform_dialect::MapNestedForallToGpuThreadsOp::applyToOne(
   DiagnosedSilenceableFailure diag =
       mlir::transform::gpu::mapNestedForallToThreadsImpl(
           rewriter, transformOp, target, getWorkgroupDims(), getSubgroupSize(),
-          true);
+          /*syncAfterDistribute=*/false);
```

Comment:
What about the TODO above?
```c++
 // TODO: synchronizations for imperfectly nested stuff.
 ```

Reply:
Not sure what that means and seems unrelated. This sync was introduced during an integrate.